docs/source/AdministratorGuide/Tutorials/basicTutoSetup.rst

```diff
@@ -0,0 +1,731 @@
+
+
+
+
+Next, edit the `/etc/init/runsvdir.conf` file to point to the future DIRAC installation as such::
+
+  # for runit - manage /usr/sbin/runsvdir-start
+  start on runlevel [2345]
+  stop on runlevel [^2345]
+  normal exit 0 111
+  respawn
+  exec /opt/dirac/sbin/runsvdir-start
+
+Finally, create the directory `/opt/dirac/sbin`::
+
+  mkdir -p /opt/dirac/sbin
+
+and the file `/opt/dirac/sbin/runsvdir-start` with the following content::
+
+  cd /opt/dirac
+  RUNSVCTRL='/sbin/runsvctrl'
+  chpst -u dirac $RUNSVCTRL d /opt/dirac/startup/*
+  killall runsv svlogd
+  RUNSVDIR='/sbin/runsvdir'
+  exec chpst -u dirac $RUNSVDIR -P /opt/dirac/startup 'log:  DIRAC runsv'
+
+make it executable::
+
+  chmod +x /opt/dirac/sbin/runsvdir-start
+
+
+and restart `runsvdir`::
+
+  restart runsvdir
+
+
+-------------
+Install MySQL
+-------------
+
+First of all, remove the existing (outdated) installation::
+
+   yum remove -y $(rpm -qa | grep -i mysql | paste -sd ' ')
+
+
+Install all the necessary RPMs for MySQL 5.7::
+
+  yum install -y https://dev.mysql.com/get/Downloads/MySQL-5.7/mysql-community-devel-5.7.25-1.el6.x86_64.rpm https://dev.mysql.com/get/Downloads/MySQL-5.7/mysql-community-server-5.7.25-1.el6.x86_64.rpm https://dev.mysqlom/get/Downloads/MySQL-5.7/mysql-community-client-5.7.25-1.el6.x86_64.rpm  https://dev.mysql.com/get/Downloads/MySQL-5.7/mysql-community-libs-5.7.25-1.el6.x86_64.rpm https://dev.mysql.com/get/Downloads/MySQL-5.7/mysql-community-common-5.7.25-1.el6.x86_64.rpm
+
+
+Setup the root password::
+
+  [root@dirac-tuto ~]# mysqld_safe --skip-grant-tables &
+  [1] 8840
+  [root@dirac-tuto ~]# 190410 16:11:21 mysqld_safe Logging to '/var/lib/mysql/dirac-tuto.err'.
+  190410 16:11:21 mysqld_safe Starting mysqld daemon with databases from /var/lib/mysql
+
+  [root@dirac-tuto ~]# mysql -u root
+  Welcome to the MySQL monitor.  Commands end with ; or \g.
+  Your MySQL connection id is 1
+  Server version: 5.6.43 MySQL Community Server (GPL)
+
+  Copyright (c) 2000, 2019, Oracle and/or its affiliates. All rights reserved.
+
+  Oracle is a registered trademark of Oracle Corporation and/or its
+  affiliates. Other names may be trademarks of their respective
+  owners.
+
+  Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.
+
+  mysql> FLUSH PRIVILEGES;
+  Query OK, 0 rows affected (0.00 sec)
+
+
+  mysql> SET PASSWORD FOR 'root'@'localhost' = PASSWORD('password');
+  Query OK, 0 rows affected (0.00 sec)
+
+  mysql> FLUSH PRIVILEGES;
+  Query OK, 0 rows affected (0.00 sec)
+
+  mysql> quit
+  Bye
+
+  [root@dirac-tuto ~]# service mysqld stop
+  Shutting down MySQL..190410 16:12:52 mysqld_safe mysqld from pid file /var/lib/mysql/dirac-tuto.pid ended
+                                                            [  OK  ]
+  [1]+  Done                    mysqld_safe --skip-grant-tables
+  [root@dirac-tuto ~]# service mysqld start
+  Starting MySQL.
+
```

Comment:
Why do you need to restart MySQL?

Reply:
Because it was started with the `--skip-grant-tables` option and needs to be restarted as default configuration